.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     resource_class: large
     docker:
       - image: cimg/node:20.12
-      - image: postgres:15-alpine
+      - image: postgres:15-bullseye
```

Comment:
Do we really need to change pg image as well?

Reply:
Didn't really dig into it, most likely it is not necessary, but just thought there could be compat issues. I can check with Ole-Martin if he sees any potential issue.

Reply:
Nah, it can stay alpine. Though changing it doesn't really matter.

Reply:
Awesome, let's keep unchanged in that case. Thank you!